### PR TITLE
[Forms][API][Web]-[SIMS-#696]Ministry Edit location and refactor - Part 2

### PR DESCRIPTION
**Pull request by guru-aot**
I have mentioned the part where I have done my changes in the form institutionlocation.json, search for my comments "Changes as part of my change" . Other tags are auto generated stuffs.

Refactoring vue imports from all the vue files.

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/institutionlocation.json [hunk 1]
```diff
@@ -112,7 +109,27 @@
                 "when": null,
                 "eq": ""
             },
-            "logic": [],
+            "logic": [
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/institutionlocation.json:112]: Changes as part of my change

File: sources/packages/web/src/views/aest/AESTDashboard.vue [hunk 2]
```diff
@@ -4,8 +4,5 @@
   </v-container>
 </template>
 <script lang="ts">
-import formio from "@/components/generic/formio.vue";
-export default {
-  components: { formio },
-};
+export default {};
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/aest/AESTDashboard.vue:7]: Can we please remove this line also?

File: sources/packages/web/src/views/institution/InstitutionDashboard.vue [hunk 3]
```diff
@@ -5,10 +5,7 @@
 </template>
 
 <script lang="ts">
-import formio from "@/components/generic/formio.vue";
-export default {
-  components: { formio },
-};
+export default {};
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/InstitutionDashboard.vue:8]: Can we please remove the script and style empty tags?

**Review by andrewsignori-aot — COMMENTED** [review]
Thanks so much for doing the cleanup, nice work. Please adjust the form.io definition as we talked 😉

File: sources/packages/forms/src/form-definitions/institutionlocation.json [hunk 4]
```diff
@@ -2372,6 +2308,79 @@
...
+            "customClass": "",
+            "modalEdit": false,
+            "defaultValue": null,
+            "persistent": "client-only",
+            "protected": false,
+            "dbIndex": false,
+            "encrypted": false,
+            "redrawOn": "data",
+            "customDefaultValue": "",
+            "calculateValue": "value = data.clientType;",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/institutionlocation.json:2324]: Do we need to do this manual association here?

File: sources/packages/forms/src/form-definitions/institutionlocation.json [hunk 5]
```diff
@@ -112,7 +112,27 @@
                 "when": null,
                 "eq": ""
             },
-            "logic": [],
+            "logic": [
+                {
+                    "name": "Client not AEST",
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/institutionlocation.json:117]: The logic name can be more close to it's purpose. like "Disable inputs for non AEST Client".

File: sources/packages/forms/src/form-definitions/institutionlocation.json (hunk 4 above)
Comment (guru-aot) [reply sources/packages/forms/src/form-definitions/institutionlocation.json:2324]: you are right, we may not need it

File: sources/packages/forms/src/form-definitions/institutionlocation.json (hunk 5 above)
Comment (guru-aot) [reply sources/packages/forms/src/form-definitions/institutionlocation.json:117]: The logic name is Client not AEST, nut the action is Disable inputs

image

Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/institutionlocation.json:117]: Actually the logic is trigger + action.

That is the name for whole logic, not trigger or action independantly.

**Review by dheepak-aot — APPROVED** [review]
Great work cleaning up. @guru-aot 👍

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍 Thanks again for the reactors ❤️

**Review by ann-aot — APPROVED** [review]
👍

**Pull request merged**